File: tp/2-torneio/Cpp/main.cpp
#include <iostream>
using namespace std;

int main(void) {

    char *partidas = new char[6];
    int v = 0;       //v = vitorias

    for(int i = 0; i < 6; i++) {
        cin >> partidas[i];
        
        if(partidas[i] == 'v' || partidas[i] == 'V') v++;
    }

    if(v > 4) cout << "3\n";
    else if(v > 2 && v <= 4) cout << "2\n";
    else if(v > 0 && v <= 2) cout << "1\n";
    else cout << "-1\n";

    return 0;
}

File: tp/2-torneio/Java/Main.java
import java.util.Scanner;

public class Main
{
	public static void main(String[] args) {
	    
	    Scanner scan = new Scanner(System.in);
	    
	    char partidas[] = new char[6];
        int v = 0;       //v = vitorias
    
        for(int i = 0; i < 6; i++) {
            partidas[i] = scan.next().charAt(0);
            
            if(partidas[i] == 'v' || partidas[i] == 'V') v++;
        }
    
        if(v > 4) System.out.println("3");
        else if(v > 2 && v <= 4) System.out.println("2");
        else if(v > 0 && v <= 2) System.out.println("1");
        else System.out.println("-1");

	    
	    scan.close();
		
	}
}
